js/main.js

```diff
@@ -6,43 +6,52 @@ window.onload = function() {
 
 function setupPlayer() {
 	var conf = {
-		key : "<YOUR_PLAYER_KEY>",
-		source : {
-			// AVC Stream
-			//dash : "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd",
-			// HEVC Stream
-			//dash : "https://bitmovin-a.akamaihd.net/content/multi-codec/hevc/stream.mpd"
-		
-			//Widevine Stream
-			dash: "http://bitmovin-a.akamaihd.net/content/art-of-motion_drm/mpds/11331.mpd",
-			drm: {
-				widevine: {
-					LA_URL: "https://widevine-proxy.appspot.com/proxy",
-				}			
-			}
-		},
+		key : "18ca6ad5-9768-4129-bdf6-17685e0d14d2",
```

Comment:
Please do not commit player_keys or other license keys to public repos. Please delete / edit the commit that we don't have it in the git history.

Reply:
fixed, sorry about that, commit history should be fixed